asserts/ifacedecls_test.go

```diff
@@ -1063,6 +1165,22 @@ func (s *plugSlotRulesSuite) TestCompilePlugRuleErrors(c *C) {
   allow-auto-connection:
     on-model:
       - foo/!qz`, `on-model in allow-auto-connection in plug rule for interface \"iface\" contains an invalid element: \"foo/!qz"`},
+		{`iface:
+  allow-installation:
+    slots-per-plug: 1`, `allow-installation in plug rule for interface "iface" cannot specify a slots-per-plug constraint, they apply only to allow-\*connection`},
+		{`iface:
+  deny-connection:
+    slots-per-plug: 1`, `deny-connection in plug rule for interface "iface" cannot specify a slots-per-plug constraint, they apply only to allow-\*connection`},
+		{`iface:
```

Comment:
For completeness, perhaps add some 'allow-connection's?

Reply:
allow-connection is actually allowed, is just will always get the normalized values